pkg/metrics/metrics.go

```diff
@@ -39,6 +39,14 @@ var (
 			Help:      `The total number of completed JobSets`,
 		}, []string{"jobset_name"},
 	)
+
+	JobSetTerminalState = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Subsystem: constants.JobSetSubsystemName,
+			Name:      "terminal_state",
+			Help:      `The current number of JobSets in terminal state (e.g. Completed, Failed)`,
+		}, []string{"jobset_name", "namespace", "terminal_state"},
```

Comment:
We should follow the pattern for the other metrics. We combine namespace/name for label. Let’s follow that here.